kernel/src/hil/flash.rs

```diff
@@ -127,7 +127,7 @@ pub trait Flash {
         buf: &'static mut Self::Page,
     ) -> Result<(), (ReturnCode, &'static mut Self::Page)>;
 
-    /// Erase a page of flash.
+    /// Erase a page of flash by setting every byte to zero.
```

Comment:
This doesn't seem right. Normally a flash erase sets everything to 1

Reply:
https://en.wikipedia.org/wiki/Flash_memory#Block_erasure

Reply:
Ah, then my suspicion of the previous comment is confirmed. Wasn't sure anymore. :+1: